lib/ansible/modules/database/influxdb/influxdb_user.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/python
+
+# (c) 2017, Vitaliy Zhhuta <zhhuta () gmail.com>
+# insipred by Kamil Szczygiel <kamil.szczygiel () intel.com> influxdb_database module
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: influxdb_user
+short_description: Manage InfluxDB users
+description:
+    - Manage InfluxDB users
+version_added: 2.5
+author: "Vitaliy Zhhuta (@zhhuta)"
+requirements:
+    - "python >= 2.6"
+    - "influxdb >= 0.9"
+options:
+    hostname:
+        description:
+            - The hostname or IP address on which InfluxDB server is listening
+        required: false
+    username:
+        description:
+            - User that we want to create
+        default: None
+        required: True
+    password:
```

Comment:
please change this to `user_password` for prevent a conflict with existing influxdb modules